easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -19,8 +19,10 @@ builddependencies = [
 ]
 
 # statically link with zlib, to avoid runtime dependency on zlib
-preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
-prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
+# further, add the system library path in the rpath: this should 'harden' the
+# resulting binutils to bootstrap GCC (no trouble when other libstdc++ is build etc)
+preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a -Wl,-rpath=/lib64 -Wl,-rpath=/usr/lib64 -Wl,-rpath=/lib -Wl,-rpath=/usr/lib"'
```

Comment:
you can't combine all the `rpath` chunks in a single one?

```
-Wl,-rpath=/lib64:/usr/lib64:/lib:/usr/lib"
```

Haven't checked, but I would fine it weird if this is not supported.

This is screaming for support in framework btw. ;-)